client/extensions/wp-super-cache/state/notices/reducer.js

```diff
@@ -1,12 +1,8 @@
 /**
- * External dependencies
- */
-import { combineReducers } from 'redux';
-
-/**
  * Internal dependencies
  */
 import { createReducer } from 'state/utils';
+import { combineReducersWithPersistence } from 'state/utils';
```

Comment:
I think there was a similar use case in https://github.com/Automattic/wp-calypso/blob/master/bin/codemods/i18n-mixin-to-localize.js.